ui/src/app/settings/views/Settings.js

```diff
@@ -10,6 +10,7 @@ const Settings = () => {
   const dispatch = useDispatch();
   useEffect(() => {
     dispatch(actions.config.fetch());
+    dispatch(actions.general.fetchOsInfo());
```

Comment:
The pattern we have in place so far (with the exception of config) is to dispatch fetch requests for data where it used, so could this live in the Commissioning component?

Reply:
Done